openapi/src/paths/events.yaml

```diff
@@ -0,0 +1,410 @@
+
+/api/v2/namespaces/{namespace}/events/broadcast:
+  post:
+    tags:
+    - events
+    summary: Broadcast a cloud event to a namespace.    
+    description: |
+      This endpoint allows you to broadcast a cloud event to a specific namespace using json encoding. Cloud events are a specification for describing event data in a common way. https://github.com/cloudevents/spec
```

Comment:
Does this endpoint support binary encoding and http-header encoding? The description suggests the answer is no, but the cloudevents spec requires that we support these various technically correct formats. https://github.com/cloudevents/spec/blob/main/cloudevents/bindings/http-protocol-binding.md

We should split this into a separate ticket if necessary to not hold up the PR.

Reply:
i created DIR-1512. Should be doable to add support for this.